setup.cfg

```diff
@@ -72,7 +72,7 @@ install_requires =
     scipy>=1.4.1 ; python_version < '3.9'
     scipy>=1.5.4 ; python_version >= '3.9'
     sphinx<5  # numpydoc dependency. sphinx>=5 breaks the docs build; see https://github.com/napari/napari/pull/4915
-    superqt>=0.3.0
+    superqt>=0.4.1
```

Comment:
@psobolewskiPhD Here is bump of superqt and in your traceback, the last position is in superqt